<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reliability</class>
 <widget class="QMainWindow" name="Reliability">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agreement analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="props/resource.qrc">:/img/logo_ethowatcher.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Create agreement matrice using ethography files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Ethography</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pb_load_eto">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Load ethography </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="eto">
             <property name="title">
              <string>Ethographys loaded</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="lb_eto_1">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lb_eto_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Categories</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QWidget" name="wd_set_categories" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
             <property name="title">
              <string>Agreement matrices (AM)</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QWidget" name="wd_matriz_2" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gb_controlers">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>400</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Bootstrap controlers</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>Optimization of the AM calculation</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Original matrix</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sb_qnt_simp_2">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Bootstrap matrices</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sb_sim_boots">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_10">
                <property name="title">
                 <string>Otimization of the prevalence and bias calculation</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Max number</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sb_numero_maximo">
                   <property name="minimum">
                    <number>100000</number>
                   </property>
                   <property name="maximum">
                    <number>2000000</number>
                   </property>
                   <property name="singleStep">
                    <number>100000</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Bootstrap sample size</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sb_reamostra">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10000</number>
                   </property>
                   <property name="singleStep">
                    <number>10</number>
                   </property>
                   <property name="value">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Threads</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sb_paralel">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_14" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_25">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Seed</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sb_seed">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10000</number>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QProgressBar" name="progress_load_eto">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_creat_analisis">
             <property name="text">
              <string>Generate Report</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Create custom agreement matrice</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>1- Insert at least three behavioral categories</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>2- Digite the categorie name</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="le_categorie_text"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pb_add_categorie">
                   <property name="text">
                    <string>Add new categorie</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_create_matriz">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>3- Start create the agreement table</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Categories</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="wd_catregorias" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
             <property name="title">
              <string>Agreement matrices (AM)</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QWidget" name="wd_matriz" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_9" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>450</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>500</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_20">
                 <item>
                  <widget class="QGroupBox" name="groupBox_7">
                   <property name="title">
                    <string>After finish inserting</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_19">
                    <item>
                     <widget class="QPushButton" name="pb_finish_editing">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>4- Finish editing agreement table</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="gb_boots_controle_1">
                   <property name="maximumSize">
                    <size>
                     <width>300</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>Bootstrap controlers</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QGroupBox" name="groupBox_13">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>90</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>90</height>
                       </size>
                      </property>
                      <property name="title">
                       <string>Otimization of the prevalence and bias calculation</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_17">
                       <item>
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>Max number</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="sb_numero_maximo_3">
                         <property name="minimum">
                          <number>100000</number>
                         </property>
                         <property name="maximum">
                          <number>2000000</number>
                         </property>
                         <property name="singleStep">
                          <number>100000</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QGroupBox" name="groupBox_12">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>120</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>120</height>
                       </size>
                      </property>
                      <property name="title">
                       <string>Optimization of the AM calculation</string>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                       <item>
                        <widget class="QWidget" name="widget_8" native="true">
                         <layout class="QVBoxLayout" name="verticalLayout_24">
                          <item>
                           <widget class="QLabel" name="label_6">
                            <property name="text">
                             <string>Original matrix</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QSpinBox" name="sb_qnt_simp_medida">
                            <property name="enabled">
                             <bool>true</bool>
                            </property>
                            <property name="minimum">
                             <number>1</number>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QWidget" name="widget_15" native="true">
                         <layout class="QVBoxLayout" name="verticalLayout_26">
                          <item>
                           <widget class="QLabel" name="label_7">
                            <property name="text">
                             <string>Bootstrap matrices</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QSpinBox" name="sb_sim_boots_2">
                            <property name="enabled">
                             <bool>true</bool>
                            </property>
                            <property name="minimum">
                             <number>1</number>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_6" native="true">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>80</height>
                       </size>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_18">
                       <item>
                        <widget class="QLabel" name="label_10">
                         <property name="text">
                          <string>Bootstrap sample size</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="sb_reamostra_2">
                         <property name="minimum">
                          <number>1</number>
                         </property>
                         <property name="maximum">
                          <number>10000</number>
                         </property>
                         <property name="value">
                          <number>100</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_16" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_9">
                       <item>
                        <widget class="QWidget" name="widget_12" native="true">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>80</height>
                          </size>
                         </property>
                         <layout class="QVBoxLayout" name="verticalLayout_23">
                          <item>
                           <widget class="QLabel" name="label_12">
                            <property name="text">
                             <string>Threads</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QSpinBox" name="sb_paralel_2">
                            <property name="minimum">
                             <number>1</number>
                            </property>
                            <property name="maximum">
                             <number>100</number>
                            </property>
                            <property name="value">
                             <number>5</number>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QWidget" name="widget_13" native="true">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>80</height>
                          </size>
                         </property>
                         <layout class="QVBoxLayout" name="verticalLayout_16">
                          <item>
                           <widget class="QLabel" name="label_13">
                            <property name="text">
                             <string>Seed</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QSpinBox" name="sb_seed_2">
                            <property name="minimum">
                             <number>1</number>
                            </property>
                            <property name="maximum">
                             <number>10000</number>
                            </property>
                            <property name="value">
                             <number>1</number>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress_editados">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Generate agreement report</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="props/resource.qrc"/>
 </resources>
 <connections/>
</ui>
